test/controllers/tmf-apis/rss.js

```diff
@@ -0,0 +1,382 @@
+var nock = require('nock'),
+    proxyquire =  require('proxyquire'),
+    testUtils = require('../../utils');
+
+describe('RSS API', function() {
+
+    var config = testUtils.getDefaultConfig();
+    var SERVER = (config.appSsl ? 'https' : 'http') + '://' + config.appHost + ':' + config.endpoints.rss.port;
+
+    var getRSSAPI = function(rssClient, tmfUtils, utils) {
+        return proxyquire('../../../controllers/tmf-apis/rss', {
+            './../../config': config,
+            './../../lib/logger': testUtils.emptyLogger,
+            './../../lib/rss': rssClient,
+            './../../lib/utils': utils
+        }).rss;
+    };
+
+    beforeEach(function() {
+        nock.cleanAll();
+    });
+
+    describe('Get Permissions', function() {
+
+        //////////////////////////////////////////////////////////////////////////////////////////////
```

Comment:
This comment can be deleted as you do not use it any with similar characteristics in the rest of the code.